### PR TITLE
[chore](variable) remove enable_nereids_planner flag

### PR DESCRIPTION
**Pull request by morrySnow**

### CODE REVIEW

**Comment by morrySnow** [comment]
run buildall

**Comment by morrySnow** [comment]
run buildall

**Comment by morrySnow** [comment]
run buildall

**Comment by morrySnow** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Review by feiniaofeiafei — APPROVED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java [hunk 1]
```diff
@@ -218,7 +216,7 @@ protected void auditAfterExec(String origStmt, StatementBase parsedStmt,
     }
 
     // only throw an exception when there is a problem interacting with the requesting client
-    protected void handleQuery(MysqlCommand mysqlCommand, String originStmt) throws ConnectionException {
+    protected void handleQuery(String originStmt) throws ConnectionException {
```
Comment (zhangm365) [inline fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java:219]: Why delete the parameter `mysqlCommand` ?

**Review by zhangm365 — COMMENTED** [review]
Why delete the parameter mysqlCommand ?

File: fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java (hunk 1 above)
Comment (zhangm365) [inline fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java:219]: Why delete the parameter mysqlCommand ?

**Pull request merged**